ci-operator/config/kubevirt/kubevirt/kubevirt-kubevirt-master.yaml

```diff
@@ -0,0 +1,26 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    context_dir: hack/kubevirt-builder
+    dockerfile_path: Dockerfile
+canonical_go_repository: kubevirt.io/kubevirt
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.2"
+  namespace: ocp
+tests:
```

Comment:
after you do the `prowgen` changes, you will be able to configure your job here. Then prowgen will be able to create it